extensions/data-workspace/src/common/telemetry.ts

```diff
@@ -26,8 +26,8 @@ export enum TelemetryActions {
 	ProjectRemovedFromWorkspace = 'ProjectRemovedFromWorkspace',
 	OpeningProject = 'OpeningProject',
 	NewProjectDialogLaunched = 'NewProjectDialogLaunched',
-	OpeningWorkspace = 'OpeningWorkspace',
```

Comment:
this isn't used anymore after we removed the "Open workspace" option from the open dialog